controllers/extension-shoot-dns-service/charts/extension-shoot-dns-service/templates/configmap-imagevector-overwrite.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.imageVectorOverwrite }}
```

Comment:
You need also `checksum` annotation in the deployment template. You can check the other extensions for example.

Reply:
addressed.